pkg/schema/schema.json

```diff
@@ -0,0 +1,172 @@
+                                "type": "IP",
+                                "replacementType": "consistent"
+                            },
+                            {
+                                "type": "MAC",
+                                "replacementType": "random"
+                            }
+                        ]
+                    ],
+                    "title": "Obfuscation Schema",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/Definitions/obfuscate"
+                    }
+                },
+                "omit": {
+                    "type": "array",
+                    "title": "Omission Schema",
+                    "description": "The omission schema defines what kind of files shall not be included in the final must-gather. This can be seen as a filter and can operate on file paths or Kubernetes and OpenShift and other custom resources. Omissions are settled first in the process of obfuscating a must-gather, so its content won't be scanned and replaced.",
+                    "default": [],
+                    "examples": [
+                        [
+                            {
+                                "type": "kubernetes",
+                                "kind": "Secret"
+                            },
+                            {
+                                "type": "kubernetes",
+                                "kind": "ConfigMap",
+                                "namespaces": [
+                                    "kube-system"
+                                ]
+                            },
+                            {
+                                "type": "file",
+                                "pattern": "*.log"
+                            },
+                            {
+                                "type": "file",
+                                "pattern": "release-4.10/ingress_controllers/*/haproxy.*"
+                            }
+                        ]
+                    ],
+                    "additionalItems": true,
+                    "items": {
+                        "anyOf": [
+                            {
+                                "$ref": "#/Definitions/k8s-omission"
+                            },
+                            {
+                                "$ref": "#/Definitions/file-omission"
+                            }
+                        ]
+                    }
+                }
+            },
+            "additionalProperties": true
+        }
+    },
+    "Definitions": {
+        "obfuscate": {
+            "type": "object",
+            "required": [
+                "type",
+                "replacementType"
+            ],
+            "properties": {
+                "type": {
+                    "type": "string",
+                    "enum": [
+                        "IP",
+                        "MAC",
+                        "keywords",
```

Comment:
I meant all the enum values should be in camel case like `Keyword`, `Regex`.

Reply:
okay, so when camel then we'll do:

>    "enum": ["IP","MAC","KeyWords","RegEx"],

is that OK or confusing?

Reply:
Keyword is still one word. So it's `Keywords` or `Keyword` depending on whether you're replacing one or more keywords. The same goes for `Regex`.

Reply:
done